server/controller/user/getProfile.js

```diff
@@ -0,0 +1,21 @@
+const { getProfile } = require('../../database/queries');
+const { boomify } = require('../../utilis');
+
+const getUserProfile = async (req, res, next) => {
+  try {
+    const { userId } = req;
+    const { rows } = await getProfile(userId);
+    console.log(rows);
+    if (rows.length === 0) {
+      throw boomify(404, 'User doesn not exist');
+    }
+    res.json({
+      status: 200,
+      data: rows,
```

Comment:
suggestion: instead of sending data as an array of one object
![image](https://user-images.githubusercontent.com/32321796/113357642-fc064900-934c-11eb-9399-d93a65a46279.png)

- you can send data as an object directly 
```suggestion
      data: rows[0],
```
![image](https://user-images.githubusercontent.com/32321796/113357733-21935280-934d-11eb-845e-d7bd2f28b226.png)